packages/web/src/exporters/zipkin.ts

```diff
@@ -70,31 +70,35 @@ export class SplunkZipkinExporter implements SpanExporter {
   private readonly _onAttributesSerializing: SplunkExporterConfig['onAttributesSerializing'];
   private readonly _xhrSender: SplunkExporterConfig['xhrSender'];
   private readonly _beaconSender: SplunkExporterConfig['beaconSender'];
+  private readonly rumAccessToken: string;
 
   constructor({
     url,
     onAttributesSerializing = NOOP_ATTRIBUTES_TRANSFORMER,
     xhrSender = NATIVE_XHR_SENDER,
     beaconSender = NATIVE_BEACON_SENDER,
+    rumAccessToken,
   }: SplunkExporterConfig) {
     this.beaconUrl = url;
     this._onAttributesSerializing = onAttributesSerializing;
     this._xhrSender = xhrSender;
     this._beaconSender = beaconSender;
+    this.rumAccessToken = rumAccessToken as string;
   }
 
   export(
     spans: ReadableSpan[],
     resultCallback: (result: ExportResult) => void
   ): void {
-    const zspans = spans.map(span => this._mapToZipkinSpan(span));
+    const zspans = spans.map((span) => this._mapToZipkinSpan(span));
     const zJson = JSON.stringify(zspans);
     if (document.hidden && this._beaconSender && zJson.length <= 64000) {
       this._beaconSender(this.beaconUrl, zJson);
     } else {
       this._xhrSender!(this.beaconUrl, zJson, {
         Accept: '*/*',
         'Content-Type': 'text/plain;charset=UTF-8',
+        Authorization: this.rumAccessToken,
```

Comment:
changes to this file is not required right? KM collector cannot receive zipkin spans

Reply:
there are still some changes in this file, not required